docs/collections/_development/guidelines.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Development Guidelines
+order: 10
+---
+
+### Practical
+
+* Make sure there is an open issue for the stuff that you want to work on. When starting the development add a comment in the issue. This will allow for discussing possible solutions, and avoid having multiple people look into the same issue.
```

Comment:
Do you really insist on opening issues even for trivial fixes?

For example, you created the well document issue https://github.com/larshp/abapGit/issues/1488 but the commit message is empty. From my point of view, the issue is not so important but the commit message because when I deal with bugs and problems I run `git log` on command line and see only commit messages. You could save time creating the issue because even a PR can be considered an issue where you can discuss things.

Reply:
no, it should be rephrased, only needed for larger changes, in order not to waste somebodys time

Reply:
changed